package.json

```diff
@@ -167,7 +169,7 @@
     "lint:js:flow": "babel-node scripts/flow-check.js",
     "lint:js": "npm run lint:js:eslint && npm run lint:js:flow",
     "lint:css": "stylelint \"+(boilerplate|docs)/web_modules/**/*.css\"",
-    "lint": "npm run lint:js && npm run lint:css",
+    "lint": "npm-run-all --parallel lint:css lint:js:*",
```

Comment:
It will run `lint:js:flow && lint:js:eslint && lint:js ` => flow and eslint are run twice